### ISSUE TITLE
Some questions about Table 4

### ISSUE THREAD

**Issue by junleiz** [YangLinyi/GLUE-X#1]
Hello, thank you for your excellent work. I have some questions about the results on Tabels. For example, for RTE OOD, do you train your model on RTE and then directly evaluate on the test set of MRPC/HANS/QQP?
The test label of these dataset is not public available. Do you evaluate your model on validation set or test set? Thank you.

**Comment by junleiz** [comment]
I have tired to train a BERT-base on RTE and then directly evaluate it on the validation set of MRPC/QQP. The performance is extremelly bad (F1 is is only 6%). Is there anything wrong with my experiment? Thank you.

**Comment by YangLinyi** [comment]
Hi Leoozy,

It is strange to know that you have achieved 6% F1 on the validation set of MRPC/QQP. Because the random guess for a binary classification problem should be 50%. You may ignore that the reverse labels are contained between RTE and MRPC datasets, which should be treated with caution. In addition, when evaluating models on OOD datasets, we test them on the training plus validation sets of MRPC/QQP.
